File: app/src/main/java/com/softartdev/notecrypt/ui/main/MainView.java
package com.softartdev.notecrypt.ui.main;

import com.softartdev.notecrypt.model.Note;

import io.realm.RealmResults;

interface MainView {
    void onNote(long noteId);

    void onAddNote();

    void onUpdateNotes(RealmResults<Note> notes);
}

File: app/src/main/java/com/softartdev/notecrypt/ui/main/MainView.kt
package com.softartdev.notecrypt.ui.main

import com.softartdev.notecrypt.model.Note
import com.softartdev.notecrypt.ui.base.MvpView

interface MainView : MvpView {
    fun onAddNote()
    fun onUpdateNotes(notes: List<Note>)
}
